MacC-GoldenRatio/MacC-GoldenRatio/Sources/Model/Layout.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Layout.swift
+//  MacC-GoldenRatio
+//
+//  Created by woo0 on 2022/11/30.
+//
+
+import UIKit
+
+struct Layout {
+	// MARK: MyHomeView
```

Comment:
p2: 해당 struct에 대해 별도 인스턴스 생성 없이 프로퍼티만을 enum 처럼 사용한다면, private initialzer로 인스턴스 생성을 막는 것도 좋은 방법이라고 합니다!
```suggestion
struct Layout {
        private init() { }
	// MARK: MyHomeView
```

Reply:
다 반영 했습니다~~ 감사합니다!!